nextcloud/server
# fix: Show error message when CSRF check fails at login

* Helps with: https://github.com/nextcloud/server/issues/32538

## Summary

This is also known as the "double login"

## How to test

0. Be logged out
1. Open the login page
2. Clear browser cookies
3. Enter credentials and submit the form

master: you see the same page again, no changes
here: you see the same page again but with an error message

| Before | After |
|--------|--------|
| ![Bildschirmfoto vom 2023-10-06 10-52-06](https://github.com/nextcloud/server/assets/1374172/18b2dced-a707-4f9b-85a6-91114a3e1200) | ![Bildschirmfoto vom 2023-10-06 12-41-26](https://github.com/nextcloud/server/assets/1374172/047629b4-c354-4062-a41f-0bb6ce92bc5a) | 

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
